src/OutputWriters/output_construction.jl

```diff
@@ -46,10 +46,15 @@ function construct_output(user_output::Union{AbstractField, Reduction}, grid, us
     return construct_output(user_output, indices)
 end
 
+
```

Comment:
```suggestion
function construct_output(user_output::WindowedField, grid, user_indices, with_halos)
    if with_halos
        new_indices = restrict_to_interior.(user_output.indices, location(user_output), topology(grid), size(grid))
        return view(user_output, new_indices)
    end

    return user_output
end

```

Reply:
I've tried this and it's not working at the moment. I'm thinking if it isn't better to keep this PR as is and add a `with_halos` property to `Fields`?